blog/src/main/kotlin/com/example/blog/domain/post/service/PostService.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.blog.domain.post.service
+
+import com.example.blog.domain.post.domain.Post
+import com.example.blog.domain.post.presentation.dto.request.PostRequest
+
+import com.example.blog.domain.post.repository.PostRepository
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.UUID
+
+@Service
+@Transactional
+class PostService(
+    private val postRepository: PostRepository
+) {
+
+    fun createPost(postRequest: PostRequest): Post {
+        val dateFormat = SimpleDateFormat("yyyy-MM-dd")
+        val publishDate = dateFormat.parse(postRequest.publishDate)
+
+        val post = Post.createRequiredFields(
+            title = postRequest.title,
+            summary = postRequest.summary,
+            publishDate = publishDate
+        )
+        return postRepository.save(post)
+    }
+
+
+
+    fun getPostById(postId: UUID): Post? {
+        return postRepository.findById(postId).orElse(null)
+    }
+
+
+    fun updatePost(postId: UUID, postRequest: PostRequest): Post? {
+        val existingPost = postRepository.findById(postId).orElse(null) ?: return null
```

Comment:
postId를 postRequest에 필드값으로 안넣고 매개변수로 따로 받는 이유가 있을까요?
그리고 
```suggestion
        val post = postRepository.findByIdOrNull(postId) ?: 예외처리
```
이렇게 nullable처리를 하는게 좋을 것 같아요
코틀린은 nullable을 Optional말고 따로 처리하기때문에 findById보다 findByIdOrNull가 좋은 것 같아요

Reply:
수정했습니다 811cfb1d4cbed41afacdd408aeeda0539b1ca1d5

Reply:
e246b0cfa7d42611bcb9726297ac4e37ff5219a7 수정했습니다.